.pre-commit-config.yaml

```diff
@@ -163,7 +163,7 @@ repos:
         name: Run isort to sort imports in Python files
   # Keep version of black in sync wit blacken-docs, pre-commit-hook-names, update-common-sql-api-stubs
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 22.12.0
```

Comment:
This is not enough
Black version is now out of sync :(
See the code note just above
`  # Keep version of black in sync wit blacken-docs, pre-commit-hook-names, update-common-sql-api-stubs`